src/views/login/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+import GmailLogin from './components/gmail-login'
+import styles from './styles.css'
+
+
+class Login extends Component {
+
+  responseGoogle(googleUser) {
+    const idToken = googleUser.getAuthResponse().id_token
+    console.log({ accessToken: idToken }) // eslint-disable-line no-console
```

Comment:
Any console info that you'd like to keep on the code should be either `info/warn/error`, not `log`